x/superfluid/keeper/grpc_query.go

```diff
@@ -653,3 +633,73 @@ func (q Querier) UnpoolWhitelist(goCtx context.Context, req *types.QueryUnpoolWh
 		PoolIds: allowedPools,
 	}, nil
 }
+
+func (q Querier) filterConcentratedPositionLocks(ctx sdk.Context, positions []model.Position, isUnbonding bool) ([]types.ConcentratedPoolUserPositionRecord, error) {
+	// Query each position ID and determine if it has a lock ID associated with it.
+	// Construct a response with the position ID, lock ID, the amount of cl shares staked, and what those shares are worth in staked osmo tokens.
+	var clPoolUserPositionRecords []types.ConcentratedPoolUserPositionRecord
+	for _, pos := range positions {
+		lockId, err := q.Keeper.clk.GetLockIdFromPositionId(ctx, pos.PositionId)
+		switch err.(type) {
+		case cltypes.PositionIdToLockNotFoundError:
+			continue
+		case nil:
```

Comment:
nit: is it possible to use if / else without the default case here? I don't understand why we would need the default case

Reply:
added here https://github.com/osmosis-labs/osmosis/pull/5691/commits/64e400454a0d14b4850ab48757dd7f7928fdf3c0